stats/stats_test.go

```diff
@@ -720,7 +720,64 @@ func TestNaN(t *testing.T) {
 	}
 }
 
+func TestBucketLookUp(t *testing.T) {
+	var tests = []struct {
+		input float64 // input
+		start float64 // start
+		end   float64 // end
+
+	}{
+		{input: 11, start: 10, end: 11},
+		{input: 171, start: 160, end: 180},
+		{input: 180, start: 160, end: 180},
+		{input: 801, start: 800, end: 900},
+		{input: 900, start: 800, end: 900},
+		{input: 999, start: 900, end: 1000},
+		{input: 999, start: 900, end: 1000},
+		{input: 999.99, start: 900, end: 1000},
+		{input: 1000, start: 900, end: 1000},
+		{input: 1000.01, start: 900, end: 1000},
```

Comment:
this is wrong :-) I made you break your previously working code, sorry ! 
1000.01 should definitely be in the ] 1000, 2000 ] bucket like 1001 and 1001.1

it's off by 1

Reply:
Okay, I take it back.